ci-operator/step-registry/ipi/conf/aws/user-permissions/ipi-conf-aws-user-permissions-commands.sh

```diff
@@ -0,0 +1,290 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+trap 'rm -rf /tmp/aws_cred_output /tmp/pull-secret /tmp/min_perms/' EXIT TERM INT
+
+USER_POLICY_FILENAME="aws-permissions-policy-creds.json"
+USER_POLICY_FILE="${SHARED_DIR}/${USER_POLICY_FILENAME}"
+
+if [[ "${AWS_INSTALL_USE_MINIMAL_PERMISSIONS}" != "yes" ]]; then
+	echo "Custom AWS user with minimal permissions is disabled. Using AWS user from cluster profile."
+	exit 0
+fi
+
+function install_jq() {
+	echo "Checking/installing jq..."
+	if ! command -v jq; then
+		curl -L https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 -o /tmp/bin/jq && chmod +x /tmp/bin/jq
+	fi
+	echo "Installing jq done: $(which jq)"
+}
+install_jq
+
+RELEASE_IMAGE_INSTALL="${RELEASE_IMAGE_INITIAL:-}"
+if [[ -z "${RELEASE_IMAGE_INSTALL}" ]]; then
+	# If there is no initial release, we will be installing latest.
+	RELEASE_IMAGE_INSTALL="${RELEASE_IMAGE_LATEST:-}"
+fi
+cp ${CLUSTER_PROFILE_DIR}/pull-secret /tmp/pull-secret
+oc registry login --to /tmp/pull-secret
+ocp_version=$(oc adm release info --registry-config /tmp/pull-secret ${RELEASE_IMAGE_INSTALL} --output=json | jq -r '.metadata.version' | cut -d. -f 1,2)
```

Comment:
```suggestion
ocp_version=$(oc adm release info --registry-config /tmp/pull-secret ${RELEASE_IMAGE_INSTALL} -ojsonpath='{.metadata.version}' | cut -d. -f 1,2)
```

So that we can reduce the chance of installing `jq` as much as possible.

Reply:
Done